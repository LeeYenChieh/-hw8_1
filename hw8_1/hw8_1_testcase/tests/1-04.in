CznmYYRNVlLEfhHoEKbSz9sYCnZQDeXQPmmm9NF7ghj5BAaRl4LcdAS2 4L6
P5mJvVcBfP7PdIIhQIr7rcTxLqChgCC4pC0PDDc4cDPKZn3pBs5QzDY6rcfIT Ar7zgNm6lYRRnJ4Nsz
HXTDl2 zL5dpDwGCPkQwrj1Mx4uSsEqGK5mIC
P5mJvVcBfP7PdIIhQIr7rcTxLqChgCC4pC0PDDc4cDPKZn3pBs5QzDY6rcfIT display
Qzu oPn1kWt8yYr08nleJdRAej2titG5ZdX
wtXVvKrb5bm2edFAV2ZmZa5J85tfqSdVlSjatJe0raIKD qN5ZEaEh3WSomit
HXTDl2 ZFuEU
Qzu add scrollBar thickBlackBorder scrollBar
wtetcfRVAhWr 3Y1uByrz1tSRYVRyF8cP
yGJEdzj44lgV6M6U6cnQGFcFTuwtIQQUJfRSiCpGEuGmpen3axpuSDMy83jx8La Ifh0riJUhh0qdNu8Frsu0Lpq4iRLvrcOc10
