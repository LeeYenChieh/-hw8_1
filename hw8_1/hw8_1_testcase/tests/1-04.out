Ar7zgNm6lYRRnJ4Nsz 
